backend/fms_core/models/sample_lineage.py

```diff
@@ -27,23 +25,27 @@ def add_error(field: str, error: str):
 
         protocol_name = self.process_measurement.process.protocol.name
         if protocol_name == 'Extraction':
-            Sample = apps.get_model("fms_core", "Sample")
-            if self.child.sample_kind.name not in Sample.BIOSPECIMEN_TYPES_NA:
+            DerivedSample = apps.get_model("fms_core", "DerivedSample")
+            # There is only a single expected derived sample for the child and parent of an extraction
+            try:
+                child_derived = self.child.derived_samples.objects.get()
+                parent_derived = self.parent.derived_samples.objects.get()
```

Comment:
Doesnt get() take attributes-args?

Reply:
Here again this is context specific ... Extraction should never be from a pool or to a pool ... The derived samples in this case are expected to have a single value ... I test the alternate case in the except which should raise an error ... something is not right in this situation.

Reply:
Ok, assuming that the extraction from pool would raise there an error, that is fine